File: MyGoogleAppOne/app/src/main/java/com/hfad/mygoogleappone/MainActivity.java
package com.hfad.mygoogleappone;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    Button btnPlus;
    Button btnMinus;
    Button btnResult;
    TextView tvNumberOfPieces;
    TextView tvPriceOfPieces;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnPlus = findViewById(R.id.btnPlus);
        btnMinus = findViewById(R.id.btnMinus);
        btnResult = findViewById(R.id.btnResult);
        tvNumberOfPieces = findViewById(R.id.tvNumberOfPieces);
        tvPriceOfPieces = findViewById(R.id.tvPriceOfPieces);

        btnPlus.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
            }
        });

    }
}

File: kotlinFishermanHandbook/app/src/main/java/com/hfad/kotlinfishermanhandbook/MainActivity.kt
package com.hfad.kotlinfishermanhandbook

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.MenuItem
import android.widget.Toast
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.google.android.material.navigation.NavigationView

class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val nav_view = findViewById<NavigationView>(R.id.nav_view)
        nav_view.setNavigationItemSelectedListener(this)

        var list = ArrayList<ListItem>()

        list.add(ListItem(R.drawable.som, "Coм", "wqf wq32   fi n 44h fb oie E EABA2" ))
        list.add(ListItem(R.drawable.nalim, "Налим", "wqf wq32   fi n 44h fb oie E EABA2" ))
        list.add(ListItem(R.drawable.shuca, "Щука", "wqf wq32   fi n 44h fb oie E EABA2" ))
        list.add(ListItem(R.drawable.caras, "Карась", "wqf wq32   fi n 44h fb oie E EABA2" ))

        var rcView = findViewById<RecyclerView>(R.id.rcView)
        rcView.hasFixedSize()

        rcView.adapter = MyAdapter(list, this)
    }

    override fun onNavigationItemSelected(item: MenuItem): Boolean {
        when(item.itemId){
            R.id.id_fish -> Toast.makeText(this,"Id fish", Toast.LENGTH_SHORT).show()
            R.id.id_na -> Toast.makeText(this,"Id na", Toast.LENGTH_SHORT).show()
            R.id.id_sna -> Toast.makeText(this,"Id sna", Toast.LENGTH_SHORT).show()
            R.id.id_history -> Toast.makeText(this,"Id history", Toast.LENGTH_SHORT).show()
        }

        return true
    }


}